scripts/qemu/configs/default.xml

```diff
@@ -0,0 +1,53 @@
+<!-- SPDX-License-Identifier: BSD-2-Clause
+# Copyright (C) 2024 Intel Corporation.
+-->
+<domain type='qemu'>
+  <name>CascadeLake2Var1</name>
+  <os>
+    <type arch='x86_64' machine='pc'>hvm</type>
+  </os>
+  <vcpu placement='static'>4</vcpu>
+  <cpu>
+    <model>Cascadelake-Server</model>
+    <topology sockets='2' dies='1' cores='2' threads='1'/>
+    <numa>
+      <cell id='0' cpus='0-1' memory='1100' unit='MiB'>
+        <distances>
+          <sibling id='0' value='10'/>
+          <sibling id='1' value='20'/>
+          <sibling id='2' value='17'/>
+        </distances>
+      </cell>
+      <cell id='1' cpus='2-3' memory='1200' unit='MiB'>
+        <distances>
+          <sibling id='0' value='20'/>
+          <sibling id='1' value='10'/>
+          <sibling id='2' value='28'/>
+        </distances>
+      </cell>
+      <cell id='2' memory='1200' unit='MiB'>
+        <distances>
+          <sibling id='0' value='17'/>
+          <sibling id='1' value='28'/>
+          <sibling id='2' value='10'/>
+        </distances>
+      </cell>
+      <interconnects>
+        <latency initiator='0' target='0' type='access' value='10'/>
+        <bandwidth initiator='0' target='0' type='access' value='10485760' unit='B'/>
+        
+        <latency initiator='0' target='1' type='access' value='20'/>
+        <bandwidth initiator='0' target='1' type='access' value='5242880' unit='B'/>
+        
+        <latency initiator='0' target='2' type='access' value='16'/>
+        <bandwidth initiator='0' target='2' type='access' value='1048576' unit='B'/>
+       
```

Comment:
I think we're missing
```
        -numa hmat-lb,initiator=1,target=0,hierarchy=memory,data-type=access-latency,latency=20 \
        -numa hmat-lb,initiator=1,target=0,hierarchy=memory,data-type=access-bandwidth,bandwidth=5242880
```

Reply:
It's intentional, you cannot add those lines :)